qdyn/output.f90

```diff
@@ -661,6 +703,42 @@ subroutine ot_write(pb)
     ! Close file
     close(FID_VMAX)
 
+
+
+  !     ! [integer scalar] step, index of max slip rate
+  ! pb%objects_glob(1)%i => pb%it
+  ! pb%objects_glob(2)%i => pb%ivmax
+  ! ! [double scalar] time, potency, potency rate
+  ! pb%objects_glob(1)%s => pb%time
+  ! pb%objects_glob(2)%s => pb%pot
+  ! pb%objects_glob(3)%s => pb%pot_rate
+  ! ! [double vector] spatial coordinates
+  ! pb%objects_glob(1)%v => pb%mesh%xglob
+  ! pb%objects_glob(2)%v => pb%mesh%yglob
+  ! pb%objects_glob(3)%v => pb%mesh%zglob
+  ! ! [double vector] mechanical quantities
+  ! pb%objects_glob(4)%v => pb%v_glob
+  ! pb%objects_glob(5)%v => pb%theta_glob
+  ! pb%objects_glob(6)%v => pb%tau_glob
+  ! pb%objects_glob(7)%v => pb%dtau_dt_glob
+  ! pb%objects_glob(8)%v => pb%slip_glob
+  ! pb%objects_glob(9)%v => pb%sigma_glob
+  ! pb%objects_glob(10)%v => pb%pot_fault
+  ! pb%objects_glob(11)%v => pb%pot_rate_fault
+  ! ! CRP: [double vector] quantity related to vmax of each fault
+  ! pb%objects_glob(12)%v => pb%vmax_fault
+  ! ! [double vector] fault label
+  ! pb%objects_glob(1)%vi => pb%mesh%fault_label_glob
+  ! ! CRP: [double vector] quantity related to ivmax of each fault
+  ! pb%objects_glob(2)%vi => pb%ivmax_fault
+
+  ! If thermal pressurisation is requested, add P and T
+  ! if (pb%features%tp == 1) then
+  !   ! [double vector] pressure, temperature
+  !   pb%objects_glob(nbase+1)%v => pb%P_glob
+  !   pb%objects_glob(nbase+2)%v => pb%T_glob
+  ! endif
```

Comment:
What's happening here? Why is this all commented out?

Reply:
Sorry! Bad copy-pasting from previous trials. removed.